.github/workflows/branch.yml

```diff
@@ -73,6 +73,9 @@ jobs:
           subdirectory: fastlane
           bundle-install-path: CACHE_BUNDLER
 
+      - name: Create Google Services Json
+        run: echo '${{ secrets.GOOGLE_SERVICES_JSON }}' > /home/runner/work/droidconKE2022Android/droidconKE2022Android/app/google-services.json
+
```

Comment:
Get rid of this we will create a new workflow for deployments.

Reply:
If I remove this then the build will fail, the other way is to commit the **google-services.json** file which I didn't want to.